### ISSUE TITLE
CLI overwriting the backupBase and keeps trying...

### ISSUE THREAD

**Issue by ecotechie** [short-pixel-optimizer/shortpixel-php#17]
I created an alias for the file cmdShortpixelOptimize.php so I could run it in my filesystem...
Then I run:
```ShortPixelOptimize --apiKey=KEY --folder=. --compression=2 --resize='1200x2000/3' --backupBase=.. --verbose --recurseDepth=0 --cmyk2rgb --createWebP```

Problem is that the backupBase files keep getting backed up on every try and therefor I loose the original.

It also does not stop at 6 Passes like [MAX_RETRIES suggests](https://github.com/short-pixel-optimizer/shortpixel-php/blob/master/lib/ShortPixel.php#L14) in ShortPixel.php.

```
CLI5e3f4e8857145@2020-02-08 16:12:56 (0.00s) (M: 1,185,424) > ShortPixel CLI version 1.6.2

CLI5e3f4e8857145@2020-02-08 16:12:56 (0.00s) (M: 1,185,400) > ShortPixel Logging VERBOSE

CLI5e3f4e8857145@2020-02-08 16:12:56 (0.00s) (M: 1,189,360) > cURL version: 7.65.3
```

**Comment by short-pixel-optimizer** [comment]
@ecotechie, thanks for letting us know. Will check. What kind of filesystem do you have?

**Comment by short-pixel-optimizer** [comment]
From the code, it would appear that the image can be backed up again only if the optimization is done again for it, but normally the image, if already optimized, is marked down in .shortpixel as succeeded so it is not optimized again.

**Comment by ecotechie** [comment]
The filesystem is ext4 for Kubuntu Linux. Files do keep getting optimized a bit further, but the point is that the original should not be overwritten, right?

Seems the --resize flag is what's making the script try over and over.

**Comment by short-pixel-optimizer** [comment]
@ecotechie we fixed this issue in the 1.6.3 release, would be grateful if you let us know if works in your case too. Thanks!
